.github/workflows/regen-abidump.sh

```diff
@@ -0,0 +1,7 @@
+set -e
```

Comment:
Let's make it print out what it's doing?
```suggestion
set -ex
```